source/Octopus.Tentacle.Tests.Integration/Support/TestEnvironmentHelper.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Security.Principal;
+using System;
+using System.Linq;
+using Octopus.Tentacle.Util;
+
+namespace Octopus.Tentacle.Tests.Integration.Support
+{
+    public static class TestEnvironmentHelper
```

Comment:
The distinction between these is that 
- `CurrentFullUserName` will result in `WORKGROUP\WIN-VTD8GECL0HO$`
- `CurrentUserName` will result in `WIN-VTD8GECL0HO$`

But annoyingly, `Environment.UserName` (i.e. when not Windows) will just be the name (e.g. `WIN-VTD8GECL0HO$`). So I didn't name these properties with things like `CurrentDomainUserName` or anything like that 😑 

I would love some input into a name that helps distinguish these.

Otherwise, I could comment them?

Reply:
I think it's going to be hard to think of a succint name that conveys the different Linux vs Windows behaviour. I'd say a good comment on them would be fine.